.kokoro/trampoline.sh

```diff
@@ -34,17 +40,24 @@ if [[ $JOB_TYPE = "presubmit" ]]; then
     if [[ $COMMIT_MESSAGE = *"[ci skip]"* || $COMMIT_MESSAGE = *"[skip ci]"* ]]; then
         echo "[ci skip] found. Exiting"
     elif [[ $OS = "windows" ]]; then
-            python "${KOKORO_GFILE_DIR}/${TRAMPOLINE_SCRIPT}"
+            python "${KOKORO_GFILE_DIR}/${TRAMPOLINE_SCRIPT}" || set_failed_status
     else
-        python3 "${KOKORO_GFILE_DIR}/${TRAMPOLINE_SCRIPT}" $version
+        for version in "${versions[@]}"; do
+            (
+                python3 "${KOKORO_GFILE_DIR}/${TRAMPOLINE_SCRIPT}" $version || set_failed_status
```

Comment:
This is my bad. I was experimenting w/ trying to get the versions to run in parallel but didn't finish. This setup masks the from-within-the-container output.

Reply:
So this change should be reverted for now?

Reply:
yeah